### PR TITLE
zlib-flate: add page

### PR DESCRIPTION
**Pull request by git-em**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**
zlib-flate version 1.0

### CODE REVIEW

File: pages/common/zlib-flate.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# zlib-flate
+
+> Raw zlib compression and decompression program.
+> The zlib-flate program is part of the qpdf package.
+> More information: <https://manned.org/zlib-flate>.
+
+- Compress a file:
+
+`zlib-flate -compress < {{file}} > {{compressed.zlib}}`
```
Comment (EmilySeville7cfg) [inline pages/common/zlib-flate.md:9]:
```suggestion
`zlib-flate -compress < {{path/to/input_file}} > {{path/to/compressed.zlib}}`
```

File: pages/common/zlib-flate.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+- Uncompress a file:
+
+`zlib-flate -uncompress < {{compressed.zlib}} > {{file}}`
```
Comment (EmilySeville7cfg) [inline pages/common/zlib-flate.md:13]:
```suggestion
`zlib-flate -uncompress < {{path/to/compressed.zlib}} > {{path/to/input_file}}`
```

File: pages/common/zlib-flate.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+- Compress a file with a specified compression level [0-9]:
+
+`zlib-flate -compress={{compression_level}} < {{file}} > {{compressed.zlib}}`
```
Comment (EmilySeville7cfg) [inline pages/common/zlib-flate.md:17]:
```suggestion
`zlib-flate -compress={{compression_level}} < {{path/to/input_file}} > {{path/to/compressed.zlib}}`
```

**Review by EmilySeville7cfg — CHANGES_REQUESTED** [review]

File: pages/common/zlib-flate.md (hunk 2 above)
Comment (git-em) [reply pages/common/zlib-flate.md]: Thanks for the suggestions!
In this case i think `output_file` is more appropriate. Already updated.

**Review by EmilySeville7cfg — APPROVED** [review]

File: pages/common/zlib-flate.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+# zlib-flate
+
+> Raw zlib compression and decompression program.
+> The zlib-flate program is part of the qpdf package.
```
Comment (navarroaxel) [inline pages/common/zlib-flate.md:4]: We use this format
```suggestion
> Part of `qpdf`.
```

File: pages/common/zlib-flate.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+
+- Compress a file:
+
+`zlib-flate -compress < {{path/to/input_file}} > {{path/to/compressed.zlib}}`
+
+- Uncompress a file:
+
+`zlib-flate -uncompress < {{path/to/compressed.zlib}} > {{path/to/output_file}}`
+
+- Compress a file with a specified compression level [0-9]:
```
Comment (navarroaxel) [inline pages/common/zlib-flate.md:15]: Based on `gzip` page
```suggestion
- Compress a file with a specified compression level. 1=Fastest (Worst), 9=Slowest (Best):
```

Comment (git-em) [reply pages/common/zlib-flate.md:15]: I did some tests to check about the level 0 (which apparently means "no compression at all") and turns out using the levels (besides 0) didn't change the size of the resulting file... I don't have any VMs right now, but would be nice if someone could test this on another machine to verify if the same thing occurs

Comment (navarroaxel) [reply pages/common/zlib-flate.md:15]: Yeah, I checked `0` is just save.
```suggestion
- Compress a file with a specified compression level. 0=Fastest (Worst), 9=Slowest (Best):
```

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**